road_detection:
    searchOffset: 0.15
    distanceBetweenSearchlines: 0.1
    minLineWidthMul: 0.5
    maxLineWidthMul: 2
    findBySobel: true
    brightness_threshold: 50
    sobelThreshold: 50
    laneWidthOffsetInMeter: 0.1
    translateEnvironment: false
    useWeights: false
    renderDebugImage: true
    threads: 0
